lib/code_fund/emails.ex

```diff
@@ -1,12 +1,56 @@
 defmodule CodeFund.Emails do
-  import Bamboo.Email
+  use Bamboo.Phoenix, view: CodeFundWeb.EmailView
 
-  def contact_form_email(%{"name" => name, "email" => email, "subject" => subject, "body" => body}) do
-    new_email()
+  def contact_form_email(
+        %{
+          "name" => name,
+          "email" => email
+        } = attrs
+      ) do
+    base_email()
     |> to("team@codefund.io")
     |> from("#{name} <#{email}>")
-    |> subject(subject)
-    |> html_body(body)
-    |> text_body(body)
+    |> subject("Contact form submission by #{name}")
+    |> assign(:attrs, attrs)
+    |> render(:form_submission)
+  end
+
+  def advertiser_form_email(
+        %{
+          "first_name" => first_name,
+          "last_name" => last_name,
+          "email" => email
+        } = attrs
+      ) do
+    IO.inspect(attrs)
+
+    base_email()
+    |> to("team@codefund.io")
```

Comment:
Feels like we could move all these pipes into the base_email function and call it w/ the params map and "Advertiser" or "Publisher"